templates/web/pages/product/product_page.tt.html

```diff
@@ -163,8 +162,15 @@ <h2 class="title-1" property="food:name" itemprop="name">[% title %]</h1>
                     <!-- Display UPC code if the EAN starts with 0 -->
                         [% IF upc_code == 'defined' %]
                             <p id="barcode_paragraph">
-                                [% lang("barcode") %]:  <span id="barcode" property="food:code" itemprop="gtin13" style="speak-as:digits;">[% code %]</span> [% upc %]
+                                [% lang("barcode") %]:  <br>
+                                <span id="barcode" property="food:code" itemprop="gtin13" style="speak-as:digits;">[% code %]</span>
+                                [% upc %]
                             </p>
+                            <svg class="barcode-svg"
```

Comment:
Nice!

Could you change the display so that it is displayed only on desktop, and at the right of the barcode number, so that we don't lose vertical space? The barcode is very nice to have in order to be able to scan it with a mobile app, but we don't want to push the real content too much down.

e.g. something like that:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/00424d7d-9e85-438b-9bc2-ec4304ffc1d4)